tgmc.dme

```diff
@@ -266,15 +266,15 @@
 #include "code\game\world.dm"
 #include "code\game\area\almayer.dm"
 #include "code\game\area\area.dm"
-#include "code\game\area\bigred.dm"
+#include "code\game\area\BigRed.dm"

```

Comment:
Merge garbage

Reply:
yup how should I sort that?

Reply:
reset to previous commit, keep changes, don't commit these changes, force push